src/libraries/Native/Unix/System.Native/pal_random.c

```diff
@@ -68,6 +72,18 @@ int32_t SystemNative_GetCryptographicallySecureRandomBytes(uint8_t* buffer, int3
 
     assert(buffer != NULL);
 
+#ifdef __EMSCRIPTEN__
+    static bool sMissingBrowserCrypto;
+    if (!sMissingBrowserCrypto)
+    {
+        int32_t bff = mono_wasm_browser_crypto_getRandomValues(buffer, bufferLength);
+        if (bff == -1)
+            sMissingBrowserCrypto = true;
+        else
+            return 0;
+    }
+#endif
```

Comment:
This needs to be `#else` to meet the requirement stated by @GrabYourPitchforks

Reply:
addressed in recent commits